src/mainMenuScene.js

```diff
@@ -42,6 +42,7 @@ class MainMenuScene {
     onHighScoreActionObservable = new Observable();
     highScoreDialog;
     isTopMost = true;
+    buttons = [];
```

Comment:
since `selectedItem` refers to the currently selected Menu Item, this could be renamed. As well, because we are using `_menuGrid` as the container for these "buttons", could we make this a get property accessor that returns `this._menuGrid.children`? Here's what I'm thinking (can't remember OTOMH if it's children or `getChildren`, but I hope the idea is there):

 ```
get menuItems() {
    return this._menuGrid.children
```

Then, we could remove the possibility of later someone adding a new menu item and forgetting to push it into an array